components/Ink/examples/supplemental/InkTypicalUseSupplemental.m

```diff
@@ -130,19 +130,36 @@ - (void)setupExampleViews {
 }
 
 - (void)viewWillLayoutSubviews {
+#if defined(__IPHONE_11_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_11_0)
```

Comment:
I think we don't need these `#if defined(__IPHONE_11_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_11_0)` guards anymore.

Reply:
Oh Thanks! I will remove it before merging. Is it because we deprecated Xcode8?

Reply:
Yep, that's the reason.

Reply:
Cool. Thanks!